tests/expander-setup.spec.js

```diff
@@ -0,0 +1,72 @@
+jest.mock('@degjs/dom-event');
+
+import expander from '../src/expander.js';
+
+describe('expander inits', () => {
```

Comment:
[suggestion] Nice tests! I can think of a few additional ones for the future: expand/collapse tests (are the proper CSS classes and aria attributes added/removed when an expand/collapse happens), interaction tests (does clicking the trigger toggle expand/collapse), event tests (do the expected custom events fire at the proper times), and animation tests (does the expand/collapse still happen if animation is involved).